### ISSUE TITLE
zoom link is wrong

### ISSUE THREAD

**Issue by danielskatz** [chaoss/wg-science#4]
the zoom link in the readme is wrong. I'm also a bit unsure if it's a good idea to have an open zoom link in a page like this

**Comment by geekygirldawn** [comment]
Agreed. And also the meeting isn't at 3pm UTC. I think we should remove the meeting details from the README entirely and let people get the meeting logistics from the agenda doc or calendar the way we do in other WG READMEs (e.g., [data science WG](https://github.com/chaoss/wg-data-science?tab=readme-ov-file#how-to-join-us)

**Comment by InessaPawson** [comment]
> I'm also a bit unsure if it's a good idea to have an open zoom link in a page like this

@danielskatz I discussed your concern with @ElizabethN, the CHAOSS community manager since 2021. She doesn't recall any issues with posting Zoom links on GitHub. There was one incident of Zoombombing when someone posted a Zoom link on X. This aligns with my experiences as an event organizer and community manager.

**Comment by InessaPawson** [comment]
> I think we should remove the meeting details from the README entirely and let people get the meeting logistics from the agenda doc or calendar the way we do in other WG READMEs (e.g., [data science WG](https://github.com/chaoss/wg-data-science?tab=readme-ov-file#how-to-join-us)

@geekygirldawn Thank you for your feedback! I will bring up your suggestion at the next WG meeting.
